docker/entrypoint.sh

```diff
@@ -64,12 +64,13 @@ init_arches() {
 		setup_arches
 	fi
 
-	init_arches_projects
+	init_arches_project
 }
 
 
+# Setup Postgresql and Elasticsearch
 setup_arches() {
-	# Setup Postgresql and Elasticsearch (this deletes your existing database)
+	cd_arches_root
```

Comment:
Have you tested this with Projects?

It seems like this should be APP_FOLDER as opposed to ARCHES ROOT.